### PR TITLE
[zh] Resync eviction glossary

### PR DESCRIPTION
**Pull request by tengqm**
This PR also converts the EOL from CRLF to LF.

### CODE REVIEW

**Review by ydFu — APPROVED** [review]
/lgtm

File: content/zh-cn/docs/reference/glossary/eviction.md [hunk 1]
```diff
@@ -1,38 +1,40 @@
...
+Eviction is the process of terminating one or more Pods on Nodes.
+-->
+驱逐即终止 Node 上一个或多个 Pod 的过程。
+
+<!--more-->
+
+<!--
+There are two kinds of eviction:
+* [Node-pressure eviction](/docs/concepts/scheduling-eviction/node-pressure-eviction/)
+* [API-initiated eviction](/docs/concepts/scheduling-eviction/api-eviction/)
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/glossary/eviction.md:35]: The link has been changed.
```suggestion
* [API-initiated eviction](/docs/concepts/scheduling-eviction/api-eviction/)
```

**Review by Sea-n — COMMENTED** [review]
/approve

File: content/zh-cn/docs/reference/glossary/eviction.md (hunk 1 above)
Comment (Sea-n) [reply content/zh-cn/docs/reference/glossary/eviction.md:35]: Oh, my bad. 😅

**Pull request merged**